docs/design/autoware-interfaces/ad-api/features/cooperation.md

```diff
@@ -0,0 +1,91 @@
+# Cooperation
+
+## Related API
+
+- {{ link_ad_api('/api/planning/velocity_factors') }}
+- {{ link_ad_api('/api/planning/steering_factors') }}
+- {{ link_ad_api('/api/planning/cooperation/set_commands') }}
+- {{ link_ad_api('/api/planning/cooperation/set_policies') }}
+- {{ link_ad_api('/api/planning/cooperation/get_policies') }}
+
+## Description
+
+Request to cooperate (RTC) is a feature that the operator supports the decision in autonomous driving mode.
+The planning component manages each situation that requires decision as a scene.
+Autoware usually drives the vehicle using its own decisions, but the operator may prefer to make their decisions in experiments and complex situations.
+
+Each scene has an ID that doesn't change until the scene is completed or canceled.
+The operator can override the decision of the target scene using this ID.
+In practice, the user interface application can hides the specification of the ID and provides an abstracted interface to the operator.
+
+For example, in the situation in the diagram below, change lanes twice and turn left at the intersection.
+There are three scenes and each has a decision to change or keep the lane, turn left or wait.
+Here Autoware decides not to change lanes a second time due to the obstacle, so the vehicle will stop there.
+
+![cooperation-scenes](./cooperation/scenes.drawio.svg)
+
+## Architecture
+
+Modules that support RTC have the operator decision and cooperation policy in addition to the module decision as shown below.
+These modules use the merged decision that is determined by these values when planning vehicle behavior.
+See decisions section for details of these values.
+The cooperation policy is used when there is no operator decision and has a default value set by the system settings.
+If the module supports RTC, these information are available in [velocity factors or steering factors](./planning-factors.md) as [cooperation status](../types/autoware_adapi_v1_msgs/msg/CooperationStatus.md).
+
+![cooperation-architecture](./cooperation/architecture.drawio.svg)
+
+## Sequence
+
+Here is an example sequence that overrides the scene decision to force a lane change. This assumes the second scene in the example situation given earlier.
```

Comment:
It might be better to label the image mentioned in Overview chapter to be Fig.1 and mention it here.

Reply:
I wrote `the diagram in the architecture section` because I couldn't use captions on markdown.